### PR TITLE
added link to the new QGIS nightly for Mac OS

### PR DESCRIPTION
**Pull request by saberraz**
Nightly for MacOS is now available from:
https://lutraconsulting.github.io/qgis-mac-packager/

### CODE REVIEW

File: source/site/forusers/alldownloads.rst [hunk 1]
```diff
@@ -490,7 +490,11 @@ The stable package uses the system Python 2.7 - other distributions are not supp
 
 Other Python modules for plugins to use are available from `kyngchaos.com <http://www.kyngchaos.com/software/python>`_.
 
-.. _QGIS-macos-testing:
```
Comment (PeterPetrik) [inline source/site/forusers/alldownloads.rst]: this should be probably kept

File: source/site/getinvolved/development/roadmap.rst [hunk 2]
```diff
@@ -126,6 +126,5 @@ Platform Location
 Windows  :ref:`Weekly release candidate <QGIS-windows-weekly>` (standalone installer)
 \        :ref:`OSGeo4W <QGIS-windows-testing>`
 Linux    :ref:`Debian/Ubuntu <QGIS-debian-testing>`
-MacOS    :ref:`Mac OS <QGIS-macos-testing>`
```
Comment (PeterPetrik) [inline source/site/getinvolved/development/roadmap.rst]: this should be probably kept

File: source/site/forusers/alldownloads.rst (hunk 1 above)
Comment (DelazJ) [reply source/site/forusers/alldownloads.rst]: @saberraz indeed, you need to keep this reference!

**Comment by saberraz** [comment]
one more time!

**Comment by rduivenvoorde** [comment]
@saberraz is this still a valid PR? Or did things already change for the good?

**Comment by saberraz** [comment]
It is still a valid PR. Nothing has been decided about distribution of the packages yet. But, this link will be still valid in any case.

**Comment by rduivenvoorde** [comment]
Talking to Peter and Saber here in A Coruna, they will provide a new plan. So will close this for now.

**Pull request closed without merging**